frameworks/CSharp/aspnetcore/Benchmarks/Controllers/MultipleQueriesController.cs

```diff
@@ -11,6 +12,16 @@ namespace Benchmarks.Controllers
     [Route("mvc/queries")]
     public class MultipleQueriesController : Controller
     {
+        private readonly RavenDb _db;
+
+        public MultipleQueriesController()
+        { }
+
+        public MultipleQueriesController(RavenDb provider)
```

Comment:
Can't you implement `IDb` and use the same pattern as the other calls?

Reply:
Tried, but the model has to be different. RavenDB does not support integer keys among other small different things. We can build a dedicated controller if you are worried about the impact that extra instance variable may have on the other controllers.

Reply:
Right, that would be better. Also I can measure the impact it has on other scenarios (it can't have any) and get preliminary results with our infrastructure so you can iterate more quickly, based on your branches.

Reply:
@sebastienros Done.